### ISSUE TITLE
[BUG] MSBUILD : java.exe error JAVA0000: Type androidx.activity.ActivityViewModelLazyKt$viewModels$1 is defined multiple times

### ISSUE THREAD

**Issue by GeorgeLeithead** [CommunityToolkit/Maui#1956]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Did you read the "Reporting a bug" section on Contributing file?

- [X] I have read the "Reporting a bug" section on Contributing file: https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#reporting-a-bug

### Current Behavior

When trying to build and debug the `Sample` solution, I get the following error when trying to debug using Windows or Android (I do not have any other platform available):

2>CommunityToolkit.Maui.Sample -> C:\Dev\MAUI\CommunityToolKitMaui\samples\CommunityToolkit.Maui.Sample\bin\Debug\net8.0-windows10.0.19041.0\win10-x64\CommunityToolkit.Maui.Sample.dll
2>MSBUILD : java.exe error JAVA0000: Error in obj\Debug\net8.0-android\lp\141\jl\classes.jar:androidx/activity/ActivityViewModelLazyKt$viewModels$1.class:
2>MSBUILD : java.exe error JAVA0000: Type androidx.activity.ActivityViewModelLazyKt$viewModels$1 is defined multiple times: obj\Debug\net8.0-android\lp\141\jl\classes.jar:androidx/activity/ActivityViewModelLazyKt$viewModels$1.class, obj\Debug\net8.0-android\lp\174\jl\classes.jar:androidx/activity/ActivityViewModelLazyKt$viewModels$1.class
2>MSBUILD : java.exe error JAVA0000: Compilation failed
2>MSBUILD : java.exe error JAVA0000: java.lang.RuntimeException: com.android.tools.r8.CompilationFailedException: Compilation failed to complete, origin: obj\Debug\net8.0-android\lp\141\jl\classes.jar
2>MSBUILD : java.exe error JAVA0000: androidx/activity/ActivityViewModelLazyKt$viewModels$1.class
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.utils.S0.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:135)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.D8.main(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:5)
2>MSBUILD : java.exe error JAVA0000: Caused by: com.android.tools.r8.CompilationFailedException: Compilation failed to complete, origin: obj\Debug\net8.0-android\lp\141\jl\classes.jar:androidx/activity/ActivityViewModelLazyKt$viewModels$1.class
2>MSBUILD : java.exe error JAVA0000: 	at Version.fakeStackEntry(Version_8.2.33.java:0)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.T.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:5)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.utils.S0.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:82)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.utils.S0.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:32)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.utils.S0.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:31)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.utils.S0.b(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:2)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.D8.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:42)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.D8.b(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:13)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.D8.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:40)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.utils.S0.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:122)
2>MSBUILD : java.exe error JAVA0000: 	... 1 more
2>MSBUILD : java.exe error JAVA0000: Caused by: com.android.tools.r8.utils.b: Type androidx.activity.ActivityViewModelLazyKt$viewModels$1 is defined multiple times: obj\Debug\net8.0-android\lp\141\jl\classes.jar:androidx/activity/ActivityViewModelLazyKt$viewModels$1.class, obj\Debug\net8.0-android\lp\174\jl\classes.jar:androidx/activity/ActivityViewModelLazyKt$viewModels$1.class
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.utils.Q2.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:21)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.utils.D2.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:54)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.utils.D2.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:10)
2>MSBUILD : java.exe error JAVA0000: 	at java.base/java.util.concurrent.ConcurrentHashMap.merge(ConcurrentHashMap.java:2056)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.utils.D2.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:6)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.graph.m4$a.d(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:6)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.dex.c.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:61)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.dex.c.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:12)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.dex.c.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:9)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.D8.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:45)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.D8.d(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:17)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.D8.c(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:69)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.utils.S0.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:28)
2>MSBUILD : java.exe error JAVA0000: 	... 6 more
2>MSBUILD : java.exe error JAVA0000: Directory 'obj\Debug\net8.0-android\lp\141' is from 'androidx.activity.activity.aar'.
2>Done building project "CommunityToolkit.Maui.Sample.csproj" -- FAILED.

### Expected Behavior

To be able to build, debug and contribute.

### Steps To Reproduce

1. Open the `Sample` solution
2. Ensure that a DOTNET RESTORE has been performed
3. Ensure that a DOTNET WORKLOAD RESTORE has been performed
4. Select as the Debug target is wither Windows or Android [I was using a Pixel 6 PRO - Api 33 (Android 13.0 - API 33)]

### Link to public reproduction project repository

https://github.com/CommunityToolkit/Maui/

### Environment

```markdown
- .NET MAUI CommunityToolkit: latest from main branch, as of 2024-06-19
- OS: Windows 11 21H2 (OR Build 22000.3019)
- Visual Studio: Microsoft Visual Studio Community 2022 (64-bit) - Current Version 17.10.3
- .NET MAUI: 
.NET Workloads
Installed Workload Id      Manifest Version       Installation Source
-----------------------------------------------------------------------------------------------------
android                    34.0.113/8.0.100       SDK 8.0.300, VS 17.10.35013.160, VS 17.11.34929.205
aspire                     8.0.1/8.0.100          SDK 8.0.300, VS 17.10.35013.160
ios                        17.2.8053/8.0.100      SDK 8.0.300, VS 17.10.35013.160, VS 17.11.34929.205
maccatalyst                17.2.8053/8.0.100      SDK 8.0.300, VS 17.10.35013.160, VS 17.11.34929.205
macos                      14.2.8053/8.0.100      SDK 8.0.300, VS 17.10.35013.160
maui-windows               8.0.40/8.0.100         SDK 8.0.300, VS 17.10.35013.160, VS 17.11.34929.205
tvos                       17.2.8053/8.0.100      SDK 8.0.300, VS 17.10.35013.160

.NET SDKs:
Version      Status
----------------------------------------
3.1.100      .NET 3.1 is out of support.
3.1.426      .NET 3.1 is out of support.
5.0.103      .NET 5.0 is out of support.
5.0.203      .NET 5.0 is out of support.
5.0.214      .NET 5.0 is out of support.
5.0.408      .NET 5.0 is out of support.
6.0.101      Patch 6.0.131 is available.
6.0.202      Patch 6.0.203 is available.
6.0.302      Patch 6.0.321 is available.
6.0.402      Patch 6.0.423 is available.
6.0.423      Up to date.
7.0.410      .NET 7.0 is out of support.
8.0.206      Up to date.
8.0.300      Patch 8.0.302 is available.
8.0.302      Up to date.

.NET Runtimes:
Name                              Version      Status
--------------------------------------------------------------------------
Microsoft.AspNetCore.App          3.1.0        .NET 3.1 is out of support.
Microsoft.NETCore.App             3.1.0        .NET 3.1 is out of support.
Microsoft.WindowsDesktop.App      3.1.0        .NET 3.1 is out of support.
Microsoft.NETCore.App             3.1.15       .NET 3.1 is out of support.
Microsoft.WindowsDesktop.App      3.1.15       .NET 3.1 is out of support.
Microsoft.AspNetCore.App          3.1.32       .NET 3.1 is out of support.
Microsoft.NETCore.App             3.1.32       .NET 3.1 is out of support.
Microsoft.WindowsDesktop.App      3.1.32       .NET 3.1 is out of support.
Microsoft.AspNetCore.App          5.0.4        .NET 5.0 is out of support.
Microsoft.NETCore.App             5.0.4        .NET 5.0 is out of support.
Microsoft.WindowsDesktop.App      5.0.4        .NET 5.0 is out of support.
Microsoft.AspNetCore.App          5.0.9        .NET 5.0 is out of support.
Microsoft.NETCore.App             5.0.9        .NET 5.0 is out of support.
Microsoft.WindowsDesktop.App      5.0.9        .NET 5.0 is out of support.
Microsoft.NETCore.App             5.0.15       .NET 5.0 is out of support.
Microsoft.WindowsDesktop.App      5.0.15       .NET 5.0 is out of support.
Microsoft.NETCore.App             5.0.16       .NET 5.0 is out of support.
Microsoft.WindowsDesktop.App      5.0.16       .NET 5.0 is out of support.
Microsoft.AspNetCore.App          5.0.17       .NET 5.0 is out of support.
Microsoft.NETCore.App             5.0.17       .NET 5.0 is out of support.
Microsoft.WindowsDesktop.App      5.0.17       .NET 5.0 is out of support.
Microsoft.AspNetCore.App          6.0.1        Patch 6.0.31 is available.
Microsoft.NETCore.App             6.0.1        Patch 6.0.31 is available.
Microsoft.WindowsDesktop.App      6.0.1        Patch 6.0.31 is available.
Microsoft.NETCore.App             6.0.4        Patch 6.0.31 is available.
Microsoft.WindowsDesktop.App      6.0.4        Patch 6.0.31 is available.
Microsoft.AspNetCore.App          6.0.5        Patch 6.0.31 is available.
Microsoft.NETCore.App             6.0.5        Patch 6.0.31 is available.
Microsoft.WindowsDesktop.App      6.0.5        Patch 6.0.31 is available.
Microsoft.AspNetCore.App          6.0.7        Patch 6.0.31 is available.
Microsoft.NETCore.App             6.0.7        Patch 6.0.31 is available.
Microsoft.WindowsDesktop.App      6.0.7        Patch 6.0.31 is available.
Microsoft.NETCore.App             6.0.12       Patch 6.0.31 is available.
Microsoft.AspNetCore.App          6.0.30       Patch 6.0.31 is available.
Microsoft.NETCore.App             6.0.30       Patch 6.0.31 is available.
Microsoft.WindowsDesktop.App      6.0.30       Patch 6.0.31 is available.
Microsoft.AspNetCore.App          6.0.31       Up to date.
Microsoft.NETCore.App             6.0.31       Up to date.
Microsoft.WindowsDesktop.App      6.0.31       Up to date.
Microsoft.AspNetCore.App          7.0.19       .NET 7.0 is out of support.
Microsoft.NETCore.App             7.0.19       .NET 7.0 is out of support.
Microsoft.WindowsDesktop.App      7.0.19       .NET 7.0 is out of support.
Microsoft.AspNetCore.App          7.0.20       .NET 7.0 is out of support.
Microsoft.NETCore.App             7.0.20       .NET 7.0 is out of support.
Microsoft.WindowsDesktop.App      7.0.20       .NET 7.0 is out of support.
Microsoft.AspNetCore.App          8.0.5        Patch 8.0.6 is available.
Microsoft.NETCore.App             8.0.5        Patch 8.0.6 is available.
Microsoft.WindowsDesktop.App      8.0.5        Patch 8.0.6 is available.
Microsoft.AspNetCore.App          8.0.6        Up to date.
Microsoft.NETCore.App             8.0.6        Up to date.
Microsoft.WindowsDesktop.App      8.0.6        Up to date.
```

### Anything else?

2>CommunityToolkit.Maui.Sample -> C:\Dev\MAUI\CommunityToolKitMaui\samples\CommunityToolkit.Maui.Sample\bin\Debug\net8.0-windows10.0.19041.0\win10-x64\CommunityToolkit.Maui.Sample.dll
2>MSBUILD : java.exe error JAVA0000: Error in obj\Debug\net8.0-android\lp\141\jl\classes.jar:androidx/activity/ActivityViewModelLazyKt$viewModels$1.class:
2>MSBUILD : java.exe error JAVA0000: Type androidx.activity.ActivityViewModelLazyKt$viewModels$1 is defined multiple times: obj\Debug\net8.0-android\lp\141\jl\classes.jar:androidx/activity/ActivityViewModelLazyKt$viewModels$1.class, obj\Debug\net8.0-android\lp\174\jl\classes.jar:androidx/activity/ActivityViewModelLazyKt$viewModels$1.class
2>MSBUILD : java.exe error JAVA0000: Compilation failed
2>MSBUILD : java.exe error JAVA0000: java.lang.RuntimeException: com.android.tools.r8.CompilationFailedException: Compilation failed to complete, origin: obj\Debug\net8.0-android\lp\141\jl\classes.jar
2>MSBUILD : java.exe error JAVA0000: androidx/activity/ActivityViewModelLazyKt$viewModels$1.class
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.utils.S0.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:135)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.D8.main(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:5)
2>MSBUILD : java.exe error JAVA0000: Caused by: com.android.tools.r8.CompilationFailedException: Compilation failed to complete, origin: obj\Debug\net8.0-android\lp\141\jl\classes.jar:androidx/activity/ActivityViewModelLazyKt$viewModels$1.class
2>MSBUILD : java.exe error JAVA0000: 	at Version.fakeStackEntry(Version_8.2.33.java:0)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.T.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:5)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.utils.S0.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:82)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.utils.S0.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:32)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.utils.S0.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:31)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.utils.S0.b(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:2)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.D8.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:42)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.D8.b(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:13)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.D8.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:40)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.utils.S0.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:122)
2>MSBUILD : java.exe error JAVA0000: 	... 1 more
2>MSBUILD : java.exe error JAVA0000: Caused by: com.android.tools.r8.utils.b: Type androidx.activity.ActivityViewModelLazyKt$viewModels$1 is defined multiple times: obj\Debug\net8.0-android\lp\141\jl\classes.jar:androidx/activity/ActivityViewModelLazyKt$viewModels$1.class, obj\Debug\net8.0-android\lp\174\jl\classes.jar:androidx/activity/ActivityViewModelLazyKt$viewModels$1.class
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.utils.Q2.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:21)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.utils.D2.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:54)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.utils.D2.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:10)
2>MSBUILD : java.exe error JAVA0000: 	at java.base/java.util.concurrent.ConcurrentHashMap.merge(ConcurrentHashMap.java:2056)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.utils.D2.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:6)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.graph.m4$a.d(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:6)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.dex.c.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:61)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.dex.c.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:12)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.dex.c.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:9)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.D8.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:45)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.D8.d(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:17)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.D8.c(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:69)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.utils.S0.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:28)
2>MSBUILD : java.exe error JAVA0000: 	... 6 more
2>MSBUILD : java.exe error JAVA0000: Directory 'obj\Debug\net8.0-android\lp\141' is from 'androidx.activity.activity.aar'.
2>Done building project "CommunityToolkit.Maui.Sample.csproj" -- FAILED.

**Comment by brminnick** [comment]
Strange - it looks like a Java error on the Windows build. Feels like a tooling problem. 

Do you have everything up-to-date? Latest version of Visual Studio? Latest version of .NET 8 (VS doesn't auto-install it - you have to [download it manually](https://dotnet.microsoft.com/en-us/download/dotnet/8.0)? Latest version of .NET MAUI (VS also doesn't auto install it - you have to run `dotnet workload install maui` manually from the command line)?

**Comment by GeorgeLeithead** [comment]
@brminnick I raised it here, just-in-case it's a MCT issue, but like you, I think it's a tooling issue.

Having been caught out by 'tooling' before when working on MCT, it is something I do straight after pulling.  Restore/update workloads, make sure the matching SDK is installed and up-to-date, along with the runtimes, etc. I'm on THE latest version of VS, Windows is 100% up-to date, etc.
I did include all the versions in the `Environment` section, and if there is anything more I can supply, just ask.

**Comment by GeorgeLeithead** [comment]
@brminnick Well, it was tooling.  I checked the `global.json` file from the root of the repository, and it's:

``json
{
  "sdk": {
    "version": "8.0.203",
    "rollForward": "latestFeature",
    "allowPrerelease": false
  }
}
```

Whilst I had the patched version 8.0.206 installed, I did not specifically have 8.0.203 installed (I'm 100% sure I did previously).  So I manually downloaded and installed, and it all builds OK.

Just to try, I then uninstalled 8.0.203 and rebooted.  The MCT is still building and debugging all fine :-/

So, not sure what happened and possibly there are some 'fragments' left after uninstalling from the re-install.

So I guess we can close this issue.
